client/me/billing-history/transactions-table.jsx

```diff
@@ -81,18 +81,26 @@ class TransactionsTable extends React.Component {
 	};
 
 	serviceNameDescription = transaction => {
+		let planName = transaction.plan;
+		// I wish I could use lib/product-values#isJetpackPlan() here, but that always returns `false`
+		// since `transaction.product_slug` uses hyphens (e.g. `jetpack-premium`), whereas `isJetpackPlan()`
+		// internally uses `getPlan()`, which expects underscores (`jetpack_premium`). -- @ockham
+		if ( startsWith( transaction.product_slug, 'jetpack-' ) ) {
```

Comment:
A problematic bit with this is that upcoming charges won't have a `product_slug`, therefore it won't work for Jetpack plans in the Upcoming Charges section. For example, see:

**Billing History**
![](https://cldup.com/fL4vZSy4l9.png)

**Upcoming Charges**
![](https://cldup.com/e0q7LwsKpW.png)

I'd expect to have `Jetpack` prepended to the upcoming Jetpack plan charge as well.